### PR TITLE
send train model window with spark request  to generate dataframes

### PR DESCRIPTION
**Pull request by vansika**
# Problem
As of now, the script that creates dataframe uses config value to fetch listens. If someone wants to change the window (number of days on which data is to be trained), it is a bit inflexible since the config value should be changed on the spark cluster  

# Solution
Send the window with the request from lemmy.

### CODE REVIEW

**Comment by vansika** [comment]
@mayhem I think it is ready to be merged

File: listenbrainz/spark/request_queries.json [hunk 1]
```diff
@@ -1,57 +1,52 @@
 {
```
Comment (mayhem) [inline listenbrainz/spark/request_queries.json]: Why are there fewer handlers now? I'm generally confused by this commit....

Comment (vansika) [reply listenbrainz/spark/request_queries.json]: Right. Have just resolved the merge conflict in light with recent changes to master.

Comment (mayhem) [reply listenbrainz/spark/request_queries.json]: ok, @paramsingh  does this look ok?

**Pull request merged**